src/main/java/org/maxgamer/quickshop/Util/MsgUtil.java

```diff
@@ -50,6 +51,7 @@
     private static HashMap<UUID, LinkedList<String>> player_messages = new HashMap<>();
     private static QuickShop plugin = QuickShop.instance;
     private static YamlConfiguration potioni18n;
+    private static DecimalFormat decimalFormat = new DecimalFormat(plugin.getConfig().getString("decimal-format");
```

Comment:
Vault use double, so DecimalFormat is useless in out plugin

Reply:
Some big numbers may look better with comma. isn't it?